frontend/src/component/changeRequest/ChangeRequest/ChangeRequest.tsx

```diff
@@ -54,21 +64,28 @@ export const ChangeRequest: VFC<IChangeRequestProps> = ({
                                 condition={change.action === 'updateEnabled'}
                                 show={
                                     <ToggleStatusChange
-                                        // @ts-expect-error TODO: fix types
-                                        enabled={change?.payload?.enabled}
+                                        enabled={
+                                            (change?.payload as any)?.enabled
```

Comment:
This is a sign of incorrect modelling in IChange. IChange could be a discriminated union:
{action: 'updateEnabled', payload: {enabled: boolean}} | {action: 'someOtherAction', payload: action specific payload}. You can check model.ts in enterprise and find a type called IChangeRequestEvent